src/releasers/index.ts

```diff
@@ -51,7 +52,7 @@ export type ReleaseType =
 type Releasers = Record<ReleaseType, typeof ReleasePR>;
 
 const releasers: Releasers = {
-  go: GoYoshi, // TODO(codyoss): refactor this into a more generic go strategy.
+  go: Go,
```

Comment:
@codyoss Will this be problematic?

Reply:
I think so, honestly our release-please setup still confuses me a bit, haha. For sure our [apiary repo](https://github.com/googleapis/google-api-go-client/blob/master/.github/release-please.yml#L1) uses some special logic. Although I would be fine with that going away if it makes things easier.

I am not sure how google-cloud-go is working because the [release-type](https://github.com/googleapis/google-cloud-go/blob/80fbb0248e27fbfeacc717531c9f499d5a296b63/.github/workflows/release.yaml#L17) it references does not exist...

Reply:
@codyoss I think we should be able to switch you to `go-yoshi`?

Reply:
Switched google-cloud-go to go-yoshi in https://github.com/googleapis/google-cloud-go/pull/4029